web/src/components/MapView.tsx

```diff
@@ -110,6 +110,21 @@ const MapView = forwardRef(
       }
     }, [])
 
+    const zoomOnClusterPress = useCallback(
+      (event: MapLayerMouseEvent) => {
+        if (mapRef) {
+          const clusterFeatures = mapRef.queryRenderedFeatures(event.point)
+          if (0 in clusterFeatures && clusterFeatures[0].properties.cluster) {
+            mapRef.flyTo({
+              center: event.lngLat,
+              zoom: mapRef.getZoom() + 2,
+            })
```

Comment:
Same question as above, why don't we just use `setZoom`? https://docs.mapbox.com/mapbox-gl-js/api/map/#map#setzoom

Reply:
answered above